setup.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2019 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Setup script."""
+
+from setuptools import find_packages
+from setuptools import setup
+
+setup(
+    name="keras-cv",
+    description="Computer Vision extensions for Keras.",
+    url="https://github.com/keras-team/keras-cv",
+    author="The KerasCV authors",
+    author_email="keras-cv@google.com",
+    license="Apache License 2.0",
+    # tensorflow isn't a dependency because it would force the
+    # download of the gpu version or the cpu version.
+    # users should install it manually.
+    install_requires=[
+        "packaging",
+        "numpy",
+        "requests",
+        "scipy",
+        "tensorboard",
+        "ipython",
+        "kt-legacy",
```

Comment:
We should just list the stuff we actually use, as we go. This list should start with just `["tensorflow"]`. Numpy/Keras are automatically included when we depend onTF.

Reply:
added